### ISSUE TITLE
Proposal: Create a room with a skeleton code

### ISSUE THREAD

**Issue by tenntenn** [syumai/gpgsync#12]
I'd like to create a room with a skeleton code.
For example, following steps:

* Open gpgsync via `https://gpgsync.herokuapp.com/p/xxxxx`
* gpgsync shows the room id form
* Click the join button
* gpgsync opens playground with a skeleton code which was hosting in `https://play.golang.org/p/xxxx`

This proposal makes [gp](https://github.com/tenntenn/goplayground) can support gpgsync such as `gp sync main.go`.

**Comment by syumai** [comment]
OK, I'll implement this feature.

**Comment by syumai** [comment]
@tenntenn I've implemented this feature on #13 .

example:

```console
# Use shared content to create room
https://gpgsync.herokuapp.com/p/xXqRTAb2hu7

# Open room with shared content
https://gpgsync.herokuapp.com/rooms/someRoomID/p/xXqRTAb2hu7
```
